crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -267,29 +267,49 @@ pub fn create_test_provider() -> JsonRpcClient<HttpTransport> {
 }
 
 #[must_use]
-pub fn duplicate_directory_with_salt(src_path: String, to_be_salted: &str, salt: &str) -> TempDir {
+pub fn duplicate_contract_directory_with_salt(
+    src_path: String,
+    to_be_salted: &str,
+    salt: &str,
+) -> TempDir {
+    duplicate_directory_and_salt_file(src_path, None, to_be_salted, "src/lib.cairo", salt)
+}
+
+#[must_use]
+pub fn duplicate_directory_and_salt_file(
+    src_path: String,
+    dst_path: Option<String>,
+    code_to_be_salted: &str,
```

Comment:
```suggestion
    function_to_be_salted: &str,
```
?

Reply:
any part of the code that matches the string will be salted